docs/ami.md

```diff
@@ -71,19 +71,24 @@ $ sudo keydb-setup.sh
 ## Some FAQ’s
 
 <b>What happens if I don’t specify a user data script when I launch?</b>
```

Comment:
We recently migrated from v1 docusaurus to v2 which caused a number of issues (many are fixed but some remain). One of the issues with the formatting comes from the html syntax used here. The new docusaurus v2 uses JSX and sometimes ends up messing up formatting that used to work. The `<b> text </b>` should be replaced by `** text **` for markdown syntax. Some html still seems to work, however in general we should use markdown syntax.